### PR TITLE
Adding card components related to menu item

### PR DESCRIPTION
**Pull request by xyang6230**
Adding OrderItemCard, MenuItemDetailCard and MenuItemPreviewCard
Screenshot 2023-12-01 at 11 31 42 PM

### CODE REVIEW

File: src/components/MenuItemDetailCard.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+                alt="example"
+                src={props.image}
+            />
+        }
+        className="border-2 border-solid border-primary w-80"
+    >
+        <div className="flex flex-col">
+            <div>
+                <span className="text-primary font-heading"> {props.name} </span>
+                <p className="text-xs text-black font-normal"> {props.description} </p>
```
Comment (congruiyS2023) [inline src/components/MenuItemDetailCard.jsx]: Should be `font-parapraph` for Barlow

File: src/components/MenuItemPreviewCard.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        }
+        onClick={onClickMenuItem}
+        className="border-2 border-solid border-primary w-40"
+    >
+        <div className="flex flex-col">
+            <div>
+                <span className="text-primary font-heading"> {props.name} </span>
+            </div>
+            <div>
+                <span className="text-primary font-normal"> {props.price} </span>
```
Comment (congruiyS2023) [inline src/components/MenuItemPreviewCard.jsx]: Same here, for Barlow, please specify `className` as `font-paragraph`

**Review by congruiyS2023 — CHANGES_REQUESTED** [review]

File: src/components/MenuItemDetailCard.jsx (hunk 1 above)
Comment (xyang6230) [reply src/components/MenuItemDetailCard.jsx]: Updated

File: src/components/MenuItemPreviewCard.jsx (hunk 2 above)
Comment (xyang6230) [reply src/components/MenuItemPreviewCard.jsx]: updated

**Review by congruiyS2023 — APPROVED** [review]
Looking good! Thanks~

**Pull request merged**